corporate/tests/test_stripe.py

```diff
@@ -4418,6 +4420,31 @@ def test_is_sponsored_realm(self) -> None:
         realm.save()
         self.assertTrue(is_sponsored_realm(realm))
 
+    def test_change_remote_server_plan_type(self) -> None:
+        server_uuid = "demo-1234"
+        remote_server = RemoteZulipServer.objects.create(
+            uuid=server_uuid,
+            api_key="magic_secret_api_key",
+            hostname="demo.example.com",
+            contact_email="email@example.com",
+        )
+        self.assertEqual(remote_server.plan_type, Realm.PLAN_TYPE_SELF_HOSTED)
+
+        new_plan_type = 3
```

Comment:
I think ideally we want this to be a value from the namespace of plan types; probably we can make a placeholder `RemoteZulipServer.PLAN_TYPE_STANDARD` with a value like 102.  I'd also go with replacing `Realm.PLAN_TYPE_SELF_HOSTED` with `RemoteZulipServer.PLAN_TYPE_SELF_HOSTED = 1` for clarity that this is its own namespace.